app/utils/runtime-environment.ts

```diff
@@ -0,0 +1,10 @@
+declare const RUNTIME_ENVIRONMENT: Record<string, string>;
```

Comment:
@charlesdemers And where would be the best place/way to declare this global?

Reply:
This one is for FastBoot, right? I think you could declare it in the same file as the other, but instead of putting it inside `Window` we could put it inside `global`

Reply:
👌 This +

```js
globals: {
  RUNTIME_ENVIRONMENT: 'readonly',
},
```

for `eslint` and we’re all set!